internal/syncasync/sync_async_bridge.go

```diff
@@ -341,11 +403,8 @@ func (sa *syncAsyncBridge) handleApprovalOpFailedEvent(event *fftypes.EventDeliv
 	if err := txcommon.RetrieveTokenApprovalInputs(sa.ctx, op, &approval); err != nil {
 		log.L(sa.ctx).Warnf("Failed to extract token approval inputs for operation '%s': %s", op.ID, err)
 	}
-	// See if this is a failure of an inflight token transfer operation
-	inflight := sa.getInFlight(event.Namespace, tokenApproveConfirm, approval.LocalID)
-	if inflight != nil {
-		go sa.resolveFailedTokenTransfer(inflight, approval.LocalID)
-	}
+
+	go sa.resolveFailedTokenTransfer(inflight, approval.LocalID)
```

Comment:
I see this was preserved, but it looks wrong..?

Reply:
good spot - looks like we were missing a helper function and message for failed approvals. Added